packages/backup-restore-operator/charts/templates/s3-secret.yaml

```diff
@@ -11,18 +11,16 @@ stringData:
   {{- with .Values.s3 }}
   {{- if .credentialSecretName }}
   credentialSecretName: {{ .credentialSecretName }} 
-  {{- end }}
-  {{- if .credentialSecretNamespace }}
-  credentialSecretNamespace: {{ .credentialSecretNamespace }} 
+  credentialSecretNamespace: {{ required "When providing a Secret containing S3 credentials, a valid .Values.credentialSecretNamespace must be provided" .credentialSecretNamespace }}
```

Comment:
`credentialSecretNamespace` hasn't actually changed in values.yaml, did we mean to change `credentialSecretName ` ? Or is that not required.

Reply:
@cbron yes in values.yaml `credentialSecretNamespace` was already empty, and I wanted to set `credentialSecretName` to an empty string too. Neither of the fields are required. But if the credentialSecretName is set then the namespace should also be set. That's why I made this change